packages/validator-ajv8/src/precompiledValidator.ts

```diff
@@ -101,7 +102,8 @@ export default class AJV8PrecompiledValidator<
    * @throws - Error when the schema provided does not match the base schema of the precompiled validator
    */
   rawValidation<Result = any>(schema: S, formData?: T): RawValidationErrorsType<Result> {
-    if (!isEqual(schema, this.rootSchema)) {
+    const resolvedRootSchema = retrieveSchema(this, this.rootSchema, this.rootSchema, formData);
```

Comment:
Odd, I swear I gave feedback on this. Rather than retrieving the root schema every time perhaps do this in the constructor only once?

Reply:
yeah, I think that will work, updating now